app/cypress/e2e/cif/project-revision/create-subsequent-project-revision.cy.js

```diff
@@ -197,5 +179,7 @@ describe("the project amendment and revisions page", () => {
     cy.findByText("General Comments (optional)")
       .next()
       .contains("test change reason");
+    // Screenshot below is commented out because of flakiness. Documented in ticket #1194.
+    // cy.happoAndAxe("Project Revision View", "view", "main");
```

Comment:
I'm assuming this will be flaky because it's the same page as one above that's commented out